src/api/Comments.js

```diff
@@ -151,6 +151,7 @@ class Comments extends OffsetBasedAPI {
         file,
         commentId,
         message,
+        tagged_message,
```

Comment:
Would it be worth updating this method to accept a `comment` property containing `id`, `message`, and `tagged_message` rather than separate properties for each?

Reply:
Maybe as a separate refactor including all the other comment methods that use this pattern